Doppler.ContactPolicies.Business.Logic.Test/ContactPoliciesServiceTest.cs

```diff
@@ -36,5 +43,81 @@ public async Task
             Assert.Equal(actual.AccountName, expected.AccountName);
             Assert.True(actual.Active);
         }
+
+        [Fact]
+        public async Task
+            UpdateContactPoliciesSettings_Should_InvokeContactPoliciesSettingsRepository_With_ProperParameters()
+        {
+            // Arrange
+            var idUser = 1000;
+
+            var excludedLists = new List<ExcludedSubscribersLists>
+            {
+                new ExcludedSubscribersLists() { Id = 1, Name = "Test" }
+            };
+            var timeRestrictionDto = new ContactPoliciesTimeRestrictionDto()
+            {
+                TimeSlotEnabled = true,
+                HourFrom = 0,
+                HourTo = 23,
+                WeekdaysEnabled = false,
+            };
+            var contactPoliciesSettingDto = new ContactPoliciesSettingsDto()
+            {
+                AccountName = "prueba@makingsense.com",
+                Active = true,
+                EmailsAmountByInterval = 999,
+                IntervalInDays = 30,
+                ExcludedSubscribersLists = excludedLists,
+                TimeRestriction = timeRestrictionDto,
+            };
+
+            var contactPoliciesSettingExpected = new ContactPoliciesSettings()
+            {
+                AccountName = null,
+                Active = true,
+                EmailsAmountByInterval = 999,
+                IntervalInDays = 30,
+                UserHasContactPolicies = false,
+                ExcludedSubscribersLists = excludedLists,
+            };
+            var contactPoliciesTimeRestrictionExpected = new ContactPoliciesTimeRestriction()
+            {
+                TimeSlotEnabled = true,
+                HourFrom = 0,
+                HourTo = 23,
+                WeekdaysEnabled = false,
+            };
```

Comment:
This could be created with _Fixture_

Reply:
Done: https://github.com/FromDoppler/doppler-contact-policies-api/pull/180/commits/be7027cd25bf622d8c3e04f8b67603f0c48bd7c6